src/views/pages/NotFoundPage/index.ts

```diff
@@ -1,8 +1,14 @@
+import { NotFoundController } from './not-found-controller';
+import { NotFoundView } from './not-found-view';
+
 export class NotFoundPage {
-  public render(): HTMLElement {
-    const container = document.createElement('div');
-    container.innerHTML = 'NotFoundPage';
+  private readonly view: NotFoundView;
 
-    return container;
+  constructor() {
```

Comment:
Router передает appModel во все конструкторы страниц, NotFoundPage его не принимает. Чтобы было единообразно, давайте добавим appModel в конструктор, даже если он там пока не используется. Либо нужно будет менять Router чтобы он был умнее. Проще первое.

```suggestion
  constructor(private readonly appModel: AppModel) {
```